nodes/src/nodes/rerank_cohere/IInstance.py

```
@@ -0,0 +1,108 @@
+        if not question.documents:
+            raise ValueError('No documents found in question to rerank')
+
+        doc_texts = []
+        for doc in question.documents:
+            if hasattr(doc, 'page_content') and doc.page_content:
+                doc_texts.append(doc.page_content)
+            elif isinstance(doc, dict) and doc.get('page_content'):
+                doc_texts.append(doc['page_content'])
+
+        if not doc_texts:
+            raise ValueError('No document content found to rerank')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)